plugin.py

```diff
@@ -15,6 +16,30 @@ def plugin_unloaded() -> None:
     LspPyrightPlugin.cleanup()
 
 
+def deflate_dict(d: Dict[str, Any], sep: str = ".", prefix: str = "") -> Dict[str, Any]:
```

Comment:
I feel this should be a built-in `DottedDict` if this solution is preferred.

Reply:
I guess it shouldn't be needed if both ST3 and ST4 provided DottedDict through a common interface?